AOD/main_AODtrain.C

```diff
@@ -119,7 +119,7 @@ void main_AODtrain(Int_t merge=0)
 
   PrintSettings();
 
-  if (merge || doCDBconnect) {
+  if ((merge || doCDBconnect) && !gSystem->Getenv("OCDB_PATH")) {
```

Comment:
Since `OCDB_PATH` is not set for Grid jobs, nothing changes on the Grid.

Reply:
Hi Dario,
on a hopefully short time scale, the macro AODtrainRawAndMC.C will replace the two macros AODtrain.C and AODtrainsim.C, in order to have the same code to produce AODs for raw data and MC.
So, I was thinking that to avoid problems in the future, it could be worth to apply this modification also to  main_AODtrainRawAndMC.C.

Thanks!

Ciao, Francesco

Reply:
Let me update the PR. I have also noticed that AOD macros don't load the custom OCDB.